pkg/cloudprovider/aws/aws.go

```diff
@@ -175,8 +478,8 @@ func (aws *AWSCloud) Clusters() (cloudprovider.Clusters, bool) {
 }
 
 // TCPLoadBalancer returns an implementation of TCPLoadBalancer for Amazon Web Services.
-func (aws *AWSCloud) TCPLoadBalancer() (cloudprovider.TCPLoadBalancer, bool) {
-	return nil, false
+func (self *AWSCloud) TCPLoadBalancer() (cloudprovider.TCPLoadBalancer, bool) {
```

Comment:
You don't have to change them if you don't want, but I'm curious -- is there any rhyme or reason between where you're calling the class object self vs where you're calling it aws?